project-base/app/src/DataFixtures/Performance/ProductDataFixture.php

```diff
@@ -196,6 +192,8 @@ private function makeProductDataUnique(ProductData $productData)
      */
     private function getUniqueIndex()
     {
+        $this->demoDataIterationCounter++;
```

Comment:
could you please extract this into a separate commit with a descriptive commit message? I do not understand the change (the property is already increased in the `load` method, I can imagine this change is because of the variants creation but I am not sure at the moment :thinking:

Reply:
I have removed the increment in the load function. The problem was that increment did not work and I was getting errors about creating the same catnum products. I did not look into it anymore as it was not important to me and this fixed the problem. I do not see the need to create a separate commit for that.